pythia6.sh

```diff
@@ -33,6 +33,7 @@ module load BASE/1.0
 # Our environment
 setenv PYTHIA6_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
 prepend-path LD_LIBRARY_PATH \$::env(PYTHIA6_ROOT)/lib
+prepend-path AGILE_GEN_PATH \$::env(PYTHIA6_ROOT)
 $([[ ${ARCHITECTURE:0:3} == osx ]] && echo "prepend-path DYLD_LIBRARY_PATH \$::env(PYTHIA6_ROOT)/lib")
```

Comment:
Hello @qgp, I am seeing this `AGILE_GEN_PATH` too. Since it is used by AGILe, shouldn't be added in the AGILe recipe instead (and then make AGILe depend on Pythia, see #471)?